src/compiler/crystal/types.cr

```diff
@@ -2685,6 +2685,8 @@ module Crystal
   end
 
   class AnnotationType < NamedType
+    property class_types : Set(Type) = Set(Type).new
```

Comment:
Is `class_types` a good name for this? Why not call it `annotated_type` as well?

Reply:
Its a set of the classes that have a given annotation, so makes sense yea.

Reply:
=> `annotated_types`